static/allunlockables.json

```diff
@@ -17913,6 +17913,29 @@
         "IsDroppable": false,
         "Rarity": "common"
     },
+    {
```

Comment:
Potentially breaking.

Reply:
Resolved in 1f14cf4158d3291103777356b6c0715b545a37d2